### PR TITLE
Fix suggested changes

### PR DESCRIPTION
**Pull request by dolnaaa**
In this PR:
- fixes:
  - `TorpedoStore.java`

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (solt6626) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: Good typo spot by Sonar

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (solt6626) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: Why not static?

**Review by solt6626 — APPROVED** [review]

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (dolnaaa) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: because

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (dolnaaa) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: yaaaay

**Pull request merged**